release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -903,45 +860,35 @@ With this release, the logical volume manager (LVM) cluster custom resource (CR)
 
 [id="ocp-4-14-storage-standardized-sts-config"]
 ==== Standardized STS configuration workflow
-{product-title} 4.14 provides a streamlined and standardized procedure to configure Security Token Service (STS) with the AWS Elastic File Storage (EFS) Container Storage Interface (CSI) Driver Operator.
-
-For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#efs-sts_persistent-storage-csi-aws-efs[Configuring AWS EFS CSI Driver Operator with Security Token Service].
+{product-title} 4.14 provides a streamlined and standardized procedure to configure Security Token Service (STS) with the AWS Elastic File Storage (EFS) Container Storage Interface (CSI) Driver Operator. For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#efs-sts_persistent-storage-csi-aws-efs[Configuring AWS EFS CSI Driver Operator with Security Token Service].
 
 [id="ocp-4-14-storage-rwop-access-mode"]
 ==== Read Write Once Pod access mode (Technology Preview)
-{product-title} 4.14 introduces a new access mode for persistent volumes (PVs) and persistent volume claims (PVCs) called ReadWriteOncePod (RWOP), which can be used only in a single pod on a single node. This is compared to the existing ReadWriteOnce access mode where a PV or PVC can be used on a single node by many pods. This feature is supported with Technology Preview status.
-
-For more information, see xref:../storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[Access modes].
+{product-title} 4.14 introduces a new access mode for persistent volumes (PVs) and persistent volume claims (PVCs) called ReadWriteOncePod (RWOP), which can be used only in a single pod on a single node. This is compared to the existing `ReadWriteOnce` access mode where a PV or PVC can be used on a single node by many pods. This feature is supported with Technology Preview status. For more information, see xref:../storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[Access modes].
```

Comment:
I notice that the "`ReadWriteOnce` access mode" is marked up, but the "new access mode [...] called ReadWriteOncePod" is not marked up. It looks like in the [linked-to section](https://66990--docspreview.netlify.app/openshift-enterprise/latest/storage/understanding-persistent-storage#pv-access-modes_understanding-persistent-storage), none of the access modes are marked up there, FWIW. 🤷